admin/class-yoast-form.php

```diff
@@ -489,11 +489,12 @@ public function media_input( $var, $label ) {
 	/**
 	 * Create a Radio input field.
 	 *
-	 * @param string $var    The variable within the option to create the file upload field for.
-	 * @param array  $values The radio options to choose from.
-	 * @param string $label  The label to show for the variable.
+	 * @param string $var         The variable within the option to create the file upload field for.
```

Comment:
I guess this is a copy-paste comment, because it's saying the `$var` is a file upload field. It should be a `radio button`